core/hostInfo/os_c.go

```diff
@@ -49,8 +49,12 @@ func (o OSCollector) GetLabels(prefix string) (map[string]string, error) {
 		return map[string]string{}, errors.Wrap(err, constants.OSGetDataError+"failed to get OS data")
 	}
 	labels := map[string]string{
-		fmt.Sprintf("%s.%s", prefix, "docker_version"):       o.getDockerVersion(o.InfoData, false)["dockerVersion"],
-		fmt.Sprintf("%s.%s", prefix, "linux_kernel_version"): utils.SemverTrunk(osData["kernelVersion"], 2),
+		fmt.Sprintf("%s.%s", prefix, "docker_version"):                 o.getDockerVersion(o.InfoData, false)["dockerVersion"],
+		fmt.Sprintf("%s.%s%s", prefix, getOSName(), "_kernel_version"): utils.SemverTrunk(osData["kernelVersion"], 2),
+		fmt.Sprintf("%s.%s", prefix, "os"):                             getOSName(),
+	}
+	if getOSName() == "windows" {
+		labels[fmt.Sprintf("%s.healthcheck.skip", prefix)] = "true"
```

Comment:
Per our discussion with @vincent99, let's consider changing the name of this label.
Do we want to find a general pattern that would work for otherinfra services we dont want to deploy on a particular host?
Some options:

At first I thought `io.rancher.infra_services.skip=healthcheck[,serviceX]` might be nice, but the comma separated bit would mean that this could not be implemented as a simple anti-affinity label in the service definition.

So, how is `io.rancher.infra_services.deploy.healthcheck=skip`?  We could replace infra_services with just infra. And would `false` be a better value than `skip`?

@vincent99 can you give your input, since you raised some concerns on Friday about it? @alena1108 do you have any input?

Whatever we decide, do we document a label like this? Its such a specific nitch use case that I'm not sure if adding it to the general documentation would help or confuse.